semanticdb/semanticdb3/semanticdb3.md

```diff
@@ -647,41 +627,73 @@ or features from other languages.
+    <td><code>RESERVED</code></td>
+    <td>Reserved for backward compatibility.</td>
   </tr>
   <tr>
     <td><code>0x800</code></td>
-    <td><code>VARPARAM</code></td>
-    <td>Is a `var` parameter of a primary constructor?</td>
+    <td><code>RESERVED</code></td>
+    <td>Reserved for backward compatibility.</td>
   </tr>
 </table>
 
-`name`. String that represents the name of the symbol.
+`name`. String that represents the name of the corresponding definition.
+
+`location`. [Location](#location) that represents the extent of
+the corresponding definition.
 
-`location`. [Location](#location) that represents the extent of the definition of the symbol.
+`signature`. Superseded by `SymbolInformation.tpe`.
 
-`signature`. [TextDocument](#textdocument) that represents the type signature of the definition.
-In this document, `text` contains a string prettyprinted by a producer and various
-sections, e.g. [Occurrences](#symboloccurrence), contain semantic information associated
-with that string. This document does not correspond to any compilation unit and
-is created solely for the purposes of storing an attributed snippet of text.
+`members`. Superseded by `ClassInfoType.declarations` in
+`SymbolInformation.tpe`.
 
-For example, for `def x = 42`, the corresponding signature may be a document with
-`text` equal to `Int` and `occurrences` featuring an identifier with `range`
-equal to `0:0..0:3`, `symbol` equal `_root_.scala.Int#` and `role` equal to `Reference`.
+`overrides`. Symbols that are extended or overridden by this symbol
+either directly or transitively.
 
-The signature format was historically unspecified. When we got around
-to specifying the format, we found out that representing type signatures
-with documents was an evolutionary dead end. Therefore, we superseded this
-field with `tpe`.
+`tpe`. [Type](#type) that represents the definition signature.
+See [Languages](#language) for more information on which definitions have
+which signatures in supported languages.
 
-`members`. This field was historically unspecified. When we got around to
-specifying it, we superseded it with `ClassInfoType.members` in `SymbolInformation.tpe`.
+`annotation`. [Annotations](#annotation) of the corresponding definition.
 
-`overrides`. Symbols that are overridden by this symbol either directly or transitively.
+`accessibility`. [Accessibility](#accessibility) of the corresponding definition.
 
-`tpe`. [Type](#type) that represents the type signature of the definition.
+### Annotation
+
+```protobuf
+message Annotation {
+  Type tpe = 1;
+}
+```
+
+`Annotation` represents annotations. See [Languages](#languages) for
+information on how annotations in supported languages map onto this
+data structure.
+
+### Accessibility
+
+```protobuf
+message Accessibility {
+  enum Tag {
+    UNKNOWN_TAG = 0;
+    PRIVATE = 1;
+    PRIVATE_THIS = 2;
+    PRIVATE_WITHIN = 3;
+    PROTECTED = 4;
+    PROTECTED_THIS = 5;
+    PROTECTED_WITHIN = 6;
```

Comment:
No public?

Reply:
We can do `PUBLIC`. That's a great idea actually.